request_token/middleware.py

```diff
@@ -52,14 +51,16 @@ def __call__(self, request):
             "authentication middleware is installed."
         )
 
-        token = request.GET.get(JWT_QUERYSTRING_ARG)
+        if request.method == 'GET':
+            token = request.GET.get(JWT_QUERYSTRING_ARG)
+        elif request.method == 'POST':
```

Comment:
Add in POST handling. The settings `JWT_QUERYSTRING_ARG` could probably do with renaming.